Makefile

```
@@ -24,7 +17,7 @@ jupyter_notebook_port := 9005
 jupyter_python_version := 3.8
 jupyter_project_name_python_shared_object := OpenSpaceToolkitAstrodynamicsPy.cpython-38-x86_64-linux-gnu
 
-################################################################################################################################################################
+clang_format_sources_path ?= $(shell find ~+ src/ include/ test/ bindings/python/src/ -name '*.cpp' -o -name '*.cxx' -o -name '*.hpp')
```

```suggestion
clang_format_sources_path ?= $(shell find ~+ src/ include/ test/ bindings/python/src/ -name '*.cpp' -o -name '*.cxx' -o -name '*.hpp' -o -name '*.tpp')
```

to include future potential tpp file using templates pattern (added to core)